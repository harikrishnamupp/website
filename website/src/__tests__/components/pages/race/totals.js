import React from 'react'
import renderer from 'react-test-renderer'
import { useStaticQuery } from 'gatsby'
import Totals from '~components/pages/race/totals'

beforeEach(() => {
  useStaticQuery.mockImplementation(() => ({
    covidRaceDataHomepage: {
      statesReportingCases: '42',
      statesReportingDeaths: '41',
    },
  }))
})

describe('Components : Race : Totals', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Totals />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
