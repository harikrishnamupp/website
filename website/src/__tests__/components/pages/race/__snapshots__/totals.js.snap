// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Race : Totals renders correctly 1`] = `
<div
  className="totals"
>
  <h3>
    States and territories reporting race and ethnicity data
  </h3>
  <div
    className="totalsWrap"
  >
    <div
      className="total"
      role="text"
    >
      <div
        className="number"
      >
        42 states/territories
      </div>
      <div
        className="label"
      >
        Reporting positive cases
      </div>
    </div>
    <div
      className="total"
      role="text"
    >
      <div
        className="number"
      >
        41 states/territories
      </div>
      <div
        className="label"
      >
        Reporting deaths
      </div>
    </div>
  </div>
</div>
`;
